server/models/index.js

```diff
@@ -1,12 +1,32 @@
 const {Sequelize} = require('sequelize')
 const {sequelize} = require('../db')
 
-const Sauce = sequelize.define("sauces", {
-  name: Sequelize.STRING,
-  image: Sequelize.STRING,
+const Item = require('./item');
+const Category = require('./Category');
+const User = require('./User'); 
+
+Category.hasMany(Item, {
+  foreignKey: {
+    name: 'categoryId',
+    allowNull: false,
+    onUpdate: 'CASCADE',
+    onDelete: 'CASCADE'
+  }
+});
+Item.belongsTo(Category, {
+  foreignKey: {
```

Comment:
i don't think we need to define foreign key constraint on the receiving side (Item.belongsTo(Category... 

please refer to the docs:
https://sequelize.org/docs/v6/core-concepts/assocs/#options-1
Team.hasMany(Player, {
  foreignKey: 'clubId',
});
Player.belongsTo(Team);